src/redux/actions/actionCreators/login.js

```diff
@@ -40,43 +40,35 @@ export function setLoginError(loginError) {
 
 export function logout() {
```

Comment:
you call the backend point for logout so that not only the token is cleared from the backend but also be blacklisted, so that it wont be re-used in case someone logs out and stores the token somewhere and hardcodes it back to the `localStorage`

Reply:
I have made that change when someone logout, it clears everything in localStorage.